packages/common/src/openapi/openapi.json

```diff
@@ -18,7 +18,7 @@
     },
     "servers": [
         {
-            "url": "/api/v1"
+            "url": "http://localhost:8080/api/v1"
```

Comment:
do we need this hardcoded ?

Reply:
FYI  we run the api docs page in PR envs.

Reply:
It's needed to get validation to work, since it's only in dev it'll only be a problem if somebody develops on a different port